test/server/lib/security/expense.test.ts.snap

```diff
@@ -1,6 +1,25 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`lib/security/expense checkExpense() returns potential threats related to the expense payment 1`] = `
+exports[`lib/security/expense checkExpense() and checkExpenses() returns potential threats related to the expense payment (batch) 1`] = `
+"
+| scope         | level  | message                                                          |
+| ------------- | ------ | ---------------------------------------------------------------- |
+| USER          | HIGH   | This user may be impersonating multiple profiles                 |
+| USER          | MEDIUM | User is not using 2FA                                            |
+| USER          | HIGH   | User has never been paid on the platform                         |
+| PAYOUT_METHOD | HIGH   | Payout Method has never been paid on the platform                |
+| PAYOUT_METHOD | HIGH   | Payout Method details is being used by other user or collectives |"
+`;
+
+exports[`lib/security/expense checkExpense() and checkExpenses() returns potential threats related to the expense payment (batch) 2`] = `
+"
+| scope | level  | message                                  |
+| ----- | ------ | ---------------------------------------- |
+| USER  | MEDIUM | User is not using 2FA                    |
+| USER  | HIGH   | User has never been paid on the platform |"
+`;
```

Comment:
I've recently discovered [inline snapshots](https://jestjs.io/docs/snapshot-testing#inline-snapshots), they make small snapshots like this way easier to read (no need to open a separate file to get the test result). No need to change it here, it's just a FYI in case you didn't know them already